samples/check_tga/BUILD

```diff
@@ -0,0 +1,34 @@
+cc_binary(
+    name = "check_tga",
+    srcs = select({
+        "@bazel_tools//src/conditions:windows": ["src/winmain.cpp"],
+        "//conditions:default": ["src/linuxmain.cpp"],
```

Comment:
As of now, this will do, but we need to consider whether this condition is actually right for linux case. Maybe i'ts better to explicitly handle linux case. In theory, one day we might want to build for Apple.

Reply:
I'll leave a `TODO`